apps/hash-frontend/src/pages/shared/entity-type-page/shared/entity-type-header.tsx

```diff
@@ -44,7 +45,7 @@ export const EntityTypeHeader = ({
 
           {!isLatest && (
             <Link
-              href={latestVersionUrl}
+              href={generateLinkParameters(latestVersionUrl).href}
```

Comment:
```suggestion
              href={latestVersionUrl}
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/3572/commits/b31daf5c29c4daa2fcd9729c030f2852da744120